### DISCUSSION TITLE
v1.1.* 🌈 released

### DISCUSSION THREAD

**Discussion by JimmyDaddy** [JimmyDaddy/react-native-image-marker#170] [Announcements]
### 🛠️ Refactor
* upgrade RN and optimize API (https://github.com/JimmyDaddy/react-native-image-marker/pull/97) ([f766d05](https://github.com/JimmyDaddy/react-native-image-marker/commit/f766d0597868967966daf3668e54a9b458067781))
* `java` to `kotlin`, `ObjectC` to `Swift` ([0b91cd4](https://github.com/JimmyDaddy/react-native-image-marker/commit/0b91cd4baaf2f664f908483b225509e443f9bae7))

### 🚀 Features
* support multiple image watermarks (https://github.com/JimmyDaddy/react-native-image-marker/issues/113) ([ec73482](https://github.com/JimmyDaddy/react-native-image-marker/commit/ec73482f7f2fd8518845c19a549fc589aff28445))
* support multiple text and more style options (https://github.com/JimmyDaddy/react-native-image-marker/issues/104) ([0b91cd4](https://github.com/JimmyDaddy/react-native-image-marker/commit/0b91cd4baaf2f664f908483b225509e443f9bae7))
* text background border radius (https://github.com/JimmyDaddy/react-native-image-marker/issues/139) ([7a476ac](https://github.com/JimmyDaddy/react-native-image-marker/commit/7a476ac9ec650fa46db2efbfcf123e9ee0dba737))

### 🐛 Bug Fixes
* image scale (https://github.com/JimmyDaddy/react-native-image-marker/issues/114) ([783b2ab](https://github.com/JimmyDaddy/react-native-image-marker/commit/783b2abc36586c6f6087295682a348b6c9010d17))
* fix: fix https://github.com/JimmyDaddy/react-native-image-marker/issues/158 android release bug style not working
* textAlignment not uniform in android-ios, fixed https://github.com/JimmyDaddy/react-native-image-marker/issues/119 ([f86f7a0](https://github.com/JimmyDaddy/react-native-image-marker/commit/f86f7a0dcea16b555ddf8107c498daa21d8727cb))

**Comment by avetisyan66** [comment]
Hi I have this error exception in native call from js
